### PR TITLE
Added sort function

### PR DESCRIPTION
**Pull request by javedbartlett**
Added a drop down sort menu to modal using lodash on the client side.
Increased number of reviews per product.

### CODE REVIEW

File: client/src/components/DropDown.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  DropdownMenu,
+  DropdownItem,
+} from 'reactstrap';
+
+const DropDown = props => {
+  const [dropdownOpen, setDropdownOpen] = useState(false);
+
+  const toggle = () => setDropdownOpen(prevState => !prevState);
+
+  const { sortReviews } = props;
```
Comment (PhilipEnchin) [inline client/src/components/DropDown.jsx]: props validation!

look up prop-types on npm and install it, import it, then below your component...
```js
DropDown.propTypes = {
  productId: PropTypes.func.isRequired, // I think...
};
```

File: client/src/components/ReviewModal.jsx [hunk 2]
```diff
@@ -5,10 +5,11 @@
 /* eslint-disable react/prop-types */
 import React, { useState } from 'react';
 import StarRatingComponent from 'react-star-rating-component';
+import DropDown from './DropDown.jsx'
 import moment from 'moment';
 import '../styles.css';
 import {
-  Modal, ModalBody,
+  Modal, ModalBody, Dropdown, DropdownToggle, DropdownMenu, DropdownItem,
```
Comment (PhilipEnchin) [inline client/src/components/ReviewModal.jsx]: Lots of unused variables here... And in other spots too. The code can probably be clearned up a lot by just removing unused stuff.

Comment (javedbartlett) [reply client/src/components/ReviewModal.jsx]: Oops, I forgot to remove these variables once I move Dropdown to its own file. Thanks for catching that!

File: client/src/components/DropDown.jsx (hunk 1 above)
Comment (javedbartlett) [reply client/src/components/DropDown.jsx]: Implemented - thank you sir!

**Pull request merged**